Source/WebInspectorUI/UserInterface/Views/SearchSidebarPanel.js

```diff
@@ -42,12 +42,17 @@ WI.SearchSidebarPanel = class SearchSidebarPanel extends WI.NavigationSidebarPan
         this._inputElement = this._inputContainer.appendChild(document.createElement("input"));
         this._inputElement.type = "search";
         this._inputElement.spellcheck = false;
-        this._inputElement.addEventListener("search", this._searchFieldChanged.bind(this));
         this._inputElement.addEventListener("input", this._searchFieldInput.bind(this));
         this._inputElement.setAttribute("results", 5);
         this._inputElement.setAttribute("autosave", "inspector-search-autosave");
         this._inputElement.setAttribute("placeholder", WI.UIString("Search Resource Content"));
-
+        this._handleSearchInputThrottler = new Throttler(this._searchFieldInput.bind(this), 250);
+        this._inputElement.addEventListener("input", (event) => {
```

Comment:
so i think we actually should use `"change"`/`"keydown"` instead as we don't really want to `performSearch` until the developer presses the enter key since it's very expensive to search across the entire DOM and every resource and etc.

afaict from reading the C++, it seems like because the `incremental` attribute is _not_ set, `"search"` should only get fired when full changes occur (e.g. the developer hits enter)

as such, i think all we need to do for this PR is just this
```diff
-this._inputElement.addEventListener("search", this._searchFieldChanged.bind(this));
+this._inputElement.addEventListener("change", this._searchFieldChanged.bind(this));
```

Reply:
I changed "input" to "change", however minibrowser breaks now when clicking in the search side bar and tells me that there is an uncaught exception: null is not an object (evaluating 'child.treeOutline._rememberTreeElement')​ (at TreeOutline.js:​238:​26)​.

Reply:
Oh, I understand, changed.